examples/distributed_lock/example.py

```diff
@@ -0,0 +1,45 @@
+# ------------------------------------------------------------
```

Comment:
Can we change the name of the file? To something like "distributedLock.py" or "lock.py" or conventional way "app.py"

Reply:
Will do. I followed the example from `secret_store` for the naming but you have a point: none of the other examples follow this :) Will update to `lock.py`